gcivil-nyu-org/Wednesday-Fall2023-Team-2
# Business occupancy

### The only driver that can adjust a Business based Parking Space is the Owner of the Business
<br>
To test, copy the ID from the spot on the map
Then, login via the 'admin' page and search by that ID under Parking Space section
Click on the Parking Space and adjust the user via the dropdown
<br>
See screenshots below for reference
<br>

<img width="875" alt="Screen Shot 2023-11-13 at 10 46 50 PM" src="https://github.com/gcivil-nyu-org/Wednesday-Fall2023-Team-2/assets/143745586/9f46b3ea-aa6b-4824-849d-ee50cc17b52f">
<img width="998" alt="Screen Shot 2023-11-13 at 10 47 08 PM" src="https://github.com/gcivil-nyu-org/Wednesday-Fall2023-Team-2/assets/143745586/c23f68b4-10aa-4cd7-9b70-d4cd34dd34ac">
<img width="984" alt="Screen Shot 2023-11-13 at 10 47 22 PM" src="https://github.com/gcivil-nyu-org/Wednesday-Fall2023-Team-2/assets/143745586/bc369110-a018-4a78-95c7-a6dc7a7768ce">
